src/utils/string/index.ts

```diff
@@ -167,3 +167,29 @@ export const applyStrictRegex = (str: string) => `^${str}$`;
  * @return {string} A shortenend version of the input string.
  */
 export const shortenGithash = (str: string) => str?.substring(0, 7);
+
+/**
+ * Function that trims the middle portion of a string. ex: "EvergreenUI" -> "Ev...UI"
+ * The resulting length, if trimmed, is maxLength + 3 (due to ellipsis length).
+ * @param {string} text - Text to trim
+ * @param {number} maxLength - Max length before trimming text
+ * @return {string} The original or trimmed text.
+ */
+export const trimStringFromMiddle = (text: string, maxLength: number) => {
+  const ellipsis = "...";
```

Comment:
This is a good use case for the [ellipsis character](https://unicode-table.com/en/2026/): `…`. And since you cleverly used `ellipsis.length` below, it should be an easy change!